meeteval/wer/matching/cy_greedy_combination_matching.pyx

```diff
@@ -41,3 +41,51 @@ def cy_forward_col(
             else:
                 tmp[current, i] = min([tmp[prev, i - 1] + cost_substitution, tmp[current, i - 1] + 1, tmp[prev, i] + 1])
     return np.asarray(tmp[current]).copy()
+
+
+@cython.boundscheck(False)  # turn off bounds-checking for entire function
+@cython.wraparound(False)  # turn off negative index wrapping for entire function
+def cy_forward_col_time_constrained(
+    npy_uint64[:] column not None,
+    npy_uint64[:] a not None,
+    npy_uint64[:] b not None,
+    npy_float64[:] a_begin not None,
+    npy_float64[:] a_end not None,
+    npy_float64[:] b_begin not None,
+    npy_float64[:] b_end not None,
+    uint cost_substitution = 1,
+) -> np.ndarray:
+    """
+    Args:
+        column: The column to be updated
+        a: Sequence in column direction (make sure that `len(column) == len(a) + 1`! otherwise SEGFAULT!!)
+        b: Sequence in row direction. This function updates `column` `len(b)` times
+        tmp: Pre-allocated temporary memory. Must have the same shape as `column`
```

Comment:
Did you change the code and removed tmp from the signature?

Reply:
Yes, I forgot to remove that. The preallocation didn't improve the runtime much

Reply:
Removed